Content.Client/_Sunrise/TTS/TTSSystem.cs

```diff
@@ -32,7 +34,7 @@ public sealed class TTSSystem : EntitySystem
     private float _radioVolume;
     private int _fileIdx;
     private float _volumeAnnounce;
-
+    private bool _isQueue;
```

Comment:
```suggestion
    private bool _isQueueEnabled;
```

Так более понятно будет, за что отвечает бул